### ISSUE TITLE
[Bug]: drmu nx check doesn't throw error code when failing to validate cookie

### ISSUE THREAD

**Issue by ividyon** [Digitalroot-Valheim/Digitalroot.ModUploader#19]
### How are you playing the game?

Single Player - No Server

### What happened?

![2024-05-22_01-08-41__chrome](https://github.com/Digitalroot-Valheim/Digitalroot.ModUploader/assets/1077140/ac56ba3c-cdde-426c-b6a0-975d8f5924fc)

### To Reproduce

Try to embed it in a GitHub Action which should fail the job step if the program errors out

```
  nexus:
    needs: build
    runs-on: windows-latest
    steps:
      - name: Download artifact
        uses: actions/download-artifact@v3
        with:
          name: 'WitchyBND-SHA${{ github.sha }}'

      - name: Setup mod uploader
        run: dotnet tool install -g Digitalroot.ModUploader

      - name: Check mod uploader
        run: drmu nx check
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Digitalroot** [comment]
I can look into this.

**Comment by Digitalroot** [comment]
fixed in 1.4.0
